{"uid":"882d05ab023b2180","name":"test_add_contact_duplicate_name","fullName":"test.Contactos.test_add_contact#test_add_contact_duplicate_name","historyId":"5eecbf5265cd15a0ff0812e8eb1c17d7","time":{"start":1726762232353,"stop":1726762232552,"duration":199},"status":"skipped","statusMessage":"XFAIL This test case is expected to fail due to known issue.\n\nAssertionError","statusTrace":"setup_add_contact = ({'Espo-Authorization': 'YWRtaW46ODAwNDkwNTM=', 'X-Api-Key': '4c29fe9482f3bccb7ac0a7d04015a2e7'}, [])\n\n    @pytest.mark.regression\n    @pytest.mark.functional\n    @pytest.mark.xfail(reason=\"This test case is expected to fail due to known issue.\", condition=True)\n    def test_add_contact_duplicate_name(setup_add_contact):\n        headers, created_contacts = setup_add_contact\n        payload = PayloadContact().build_payload_add_contact(salutationName=\"Ms.\", firstName=\"Mike\",\n                                                             lastName=\"Towel\",\n                                                             name=\"Mike Towel\", emailAddress=\"one@on.as\",\n                                                             phoneNumber=\"+17945654564\",\n                                                             addressPostalCode=\"0000\", addressStreet=\"none\",\n                                                             addressState=\"none\", addressCity=\"none\", addressCountry=\"none\",\n                                                             description=\"prueba\")\n        response = EspocrmRequest().post(EndpointContacts.contact(), headers, payload)\n>       AssertionStatusCode().assert_status_code_200(response)\n\ntest/Contactos/test_add_contact.py:41: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nresponse = <Response [409]>\n\n    @staticmethod\n    def assert_status_code_200(response):\n>       assert response.status_code == 200\nE       AssertionError\n\nsrc/assertions/status_code_assertions.py:11: AssertionError","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"get_headers","time":{"start":1726762231405,"stop":1726762231405,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false},{"name":"setup_add_contact","time":{"start":1726762232353,"stop":1726762232353,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"afterStages":[{"name":"setup_add_contact::0","time":{"start":1726762232557,"stop":1726762232557,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":false,"stepsCount":0,"attachmentsCount":0,"hasContent":false}],"labels":[{"name":"tag","value":"@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"},{"name":"tag","value":"functional"},{"name":"tag","value":"regression"},{"name":"parentSuite","value":"test.Contactos"},{"name":"suite","value":"test_add_contact"},{"name":"host","value":"fv-az529-703"},{"name":"thread","value":"2160-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"test.Contactos.test_add_contact"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[],"tags":["functional","regression","@pytest.mark.xfail(reason='This test case is expected to fail due to known issue.', condition=True)"]},"source":"882d05ab023b2180.json","parameterValues":[]}